packages/e2e-tests/src/tests/userProfile.test.ts

```diff
@@ -106,11 +106,25 @@ describe('User Profile Feature', () => {
     await generalChannelOwner.sendMessage(users.owner.messages[0])
   })
 
-  it('Owner updates their profile photo', async () => {
+  it('Owner updates their profile photo with PNG', async () => {
     const menu = new UserProfileContextMenu(users.owner.app.driver)
     await menu.openMenu()
     await menu.openEditProfileMenu()
-    await menu.uploadPhoto()
+    await menu.uploadPNGPhoto()
+  })
+
+  it('Owner updates their profile photo with JPEG', async () => {
+    const menu = new UserProfileContextMenu(users.owner.app.driver)
+    await menu.openMenu()
+    await menu.openEditProfileMenu()
+    await menu.uploadJPEGPhoto()
+  })
+
+  it('Owner updates their profile photo with GIF', async () => {
```

Comment:
Could you check locally if userProfile test is passing?
* Last of "Owner updates their profile photo with..." tests uploads gif photo so if the upload suppose to be successful then assertion in "Owner's message with profile photo is visible on channel" should be updated from checking png to gif
* Could you maybe add assertion at the end of each of "Owner updates their profile photo with..." cases to make sure that e.g user do not see upload error message?

Reply:
Added assertions and testing locally now